src/contentful/ContentfulClientOptions.ts

```diff
@@ -7,4 +7,6 @@ export interface ContentfulClientOptions {
     readonly environmentId?: string;
     readonly preview?: boolean;
     readonly fetch?: any;
-}
\ No newline at end of file
+    readonly host?: string;
+    readonly headers?: object|string;
```

Comment:
headers should be more restrictive than `object`. Something like `Record<string, string>`?

Reply:
Good call.